mmpose/models/detectors/top_down_crowd.py

```diff
@@ -0,0 +1,346 @@
+                    show=False,
+                    wait_time=0,
+                    out_file=None):
+        """Draw `result` over `img`.
+
+        Args:
+            img (str or Tensor): The image to be displayed.
+            result (list[dict]): The results to draw over `img`
+                (bbox_result, pose_result).
+            kpt_score_thr (float, optional): Minimum score of keypoints
+                to be shown. Default: 0.3.
+            bbox_color (str or tuple or :obj:`Color`): Color of bbox lines.
+            pose_kpt_color (np.array[Nx3]`): Color of N keypoints.
+            pose_limb_color (np.array[Mx3]): Color of M limbs.
+            text_color (str or tuple or :obj:`Color`): Color of texts.
+            thickness (int): Thickness of lines.
+            font_scale (float): Font scales of texts.
+            win_name (str): The window name.
+            wait_time (int): Value of waitKey param.
+                Default: 0.
+            out_file (str or None): The filename to write the image.
+                Default: None.
+
+        Returns:
+            img (Tensor): Only if not `show` or `out_file`
+        """
+
+        img = mmcv.imread(img)
+        img = img.copy()
+        img_h, img_w, _ = img.shape
+
+        bbox_result = []
+        pose_result = []
+        for res in result:
+            bbox_result.append(res['bbox'])
+            pose_result.append(res['keypoints'])
+
+        if len(bbox_result) > 0:
+            bboxes = np.vstack(bbox_result)
+            # draw bounding boxes
+            mmcv.imshow_bboxes(
+                img,
+                bboxes,
+                colors=bbox_color,
+                top_k=-1,
+                thickness=thickness,
+                show=False,
+                win_name=win_name,
+                wait_time=wait_time,
+                out_file=None)
+
+            for person_id, kpts in enumerate(pose_result):
+                # draw each point on image
+                for kid, kpt in enumerate(kpts):
+                    x_coord, y_coord, kpt_score = int(kpt[0]), int(
+                        kpt[1]), kpt[2]
+                    if kpt_score > kpt_score_thr:
+                        # cv2.circle(img, (x_coord, y_coord), radius,
+                        #            pose_kpt_color, thickness)
+                        img_copy = img.copy()
+                        r, g, b = pose_kpt_color[kid]
+                        cv2.circle(img_copy, (int(x_coord), int(y_coord)),
+                                   radius, (int(r), int(g), int(b)), -1)
+                        transparency = max(0, min(1, kpt_score))
+                        cv2.addWeighted(
+                            img_copy,
+                            transparency,
+                            img,
+                            1 - transparency,
+                            0,
+                            dst=img)
+
+                # draw limbs
+                if skeleton is not None:
```

Comment:
The master branch is updated.
Please refer to the latest version, and update this file accordingly.